packages/flame_tiled/lib/src/renderable_layers/group_layer.dart

```diff
@@ -0,0 +1,37 @@
+part of '../renderable_tile_map.dart';
```

Comment:
To keep consistent with the rest of the code base, use `@internal` instead of `part of`.

Reply:
Converted them to internal classes. Had to use `tiled.` for `TileLayer` and `ImageLayer` due to name clash